lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer_test.go

```diff
@@ -295,8 +296,116 @@ func TestSubscriberdbCloudServicer(t *testing.T) {
 	assert.Equal(t, expectedToken, res.NextPageToken)
 }
 
+func TestSubscriberdbServicerWithDigest(t *testing.T) {
+	lte_test_init.StartTestService(t)
+	configurator_test_init.StartTestService(t)
+
+	// Create servicer with flat digest feature flag turned on.
```

Comment:
Nit: be consistent about whether you place a period at the end of your comments

The general rule of thumb in Orc8r code is "terminate with period" for docstrings and "don't terminate with period" for inline comments